### PR TITLE
Kfdef converter

### PR DESCRIPTION
**Pull request by gabrielwen**
A simple CLI to convert `KfDef` from v1alpha1 to v1beta1.

Relates to #3703

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4298)

### CODE REVIEW

**Comment by gabrielwen** [comment]
kubeflow/testing#480

File: bootstrap/cmd/kfdefConverter/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+A simple CLI to convert KfDef from v1alpha1 to v1beta1
+
+Usage:
+  kfdef-converter [command]
+
+Available Commands:
+  help        Help about any command
+  tov1beta1   Convert a KfDef config in v1alpha1 into v1beta1 format.
+
```
Comment (lluunn) [inline bootstrap/cmd/kfdefConverter/README.md]: probably add an example usage of kfdef-converter tov1beta1

Comment (gabrielwen) [reply bootstrap/cmd/kfdefConverter/README.md]: Added.

**Comment by gabrielwen** [comment]
@lluunn updated, could you give it a quick lgtm/approve?  need this to start migration over to v1beta1.

**Pull request merged**